integration_tests/test_runner/src/bootstrapping.rs

```diff
@@ -58,6 +62,144 @@ pub fn get_keys() -> Vec<ValidatorKeys> {
     ret
 }
 
+/// This function deploys the required contracts onto the Ethereum testnet
```

Comment:
These additions, and the entire solidity folder, are essentially a copy job of what is in Gravity with all the paths changed from /gravity to /althea

Reply:
we should probably delete the gravity related stuff so that we can stick to deploying erc20